app/src/ui/welcome/configure-git.tsx

```diff
@@ -12,9 +13,23 @@ interface IConfigureGitProps {
 
 /** The Welcome flow step to configure git. */
 export class ConfigureGit extends React.Component<IConfigureGitProps, {}> {
+  private configureGitElement = React.createRef<HTMLElement>()
+
+  public componentDidMount() {
+    if (this.configureGitElement.current == null) {
+      return
+    }
+
+    focusFirstSuitableChild(this.configureGitElement.current)
+  }
+
   public render() {
     return (
-      <div id="configure-git">
+      <section
+        ref={this.configureGitElement}
+        id="configure-git"
+        aria-label="Configure Git"
```

Comment:
Using `aria-label`... I tried `aria-labelledby` and it never would announce it. I triple checked for typos, copy/pasted to ensure. No idea why `aria-labelledby` doesn't work.